Jenkinsfile

```diff
@@ -133,6 +133,11 @@ echo "Final podspec: ${pod}"
 podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod) {
     node('coreos-assembler') { container('coreos-assembler') {
 
+        // declare this early so we can use it in Slack
+        def newBuildID
+
+        try {
```

Comment:
This was deliberate. All the Jenkins scopes would make it very easy to end up with a huge indent that would make it harder to read and hack on.

I can do this if you'd really like, but would prefer not to. :)

Reply:
Right now we don't indent this block, but we do indent the catch and the finally, which is a bit odd. How about a compromise? options:

- We could add a comment to the closing brace (`# end try {}`)
- We could put the code within the try inside a function so that the try/catch/finally is all visible on a single screen of text.

Any other ideas?

Reply:
I did the first, but I like the idea of the second (and I think it's closer to the RHCOS pipeline too). Let's aim to work towards that.

Factoring it out is something we'll want to do anyway as part of unifying the pipelines.